Frontend/voitheia-App/src/app/components/user/register/register.component.ts

```diff
@@ -16,7 +16,7 @@ export class RegisterComponent implements OnInit {
   faPaperPlane = faPaperPlane;
   termsChecked = false;
   
-  constructor(private _userService: UserService, private _navigationService: NavigationService, private _utilititesService: UtilitiesService) {
+  constructor(private _userService: UserService, public _navigationService: NavigationService, private _utilitiesService: UtilitiesService) {
```

Comment:
Same here: NavigationService should not be public

Reply:
changed to private